extract_queries_from_postgres_logs.py

```
@@ -124,7 +131,7 @@ def extract_queries_from_log(log_file):
                 "expected": ""
             }
         except:
-            print(norm_query)
+            logger.error(norm_query)
             import ipdb; ipdb.set_trace()
```

Ruff (0.11.9)

Multiple statements on one line (semicolon)

(E702)